experiments/standalone/neural_snow/analysis_tools.jl

```diff
@@ -0,0 +1,348 @@
+using Flux
```

Comment:
For now, let's move this to where it is used, and leave the Project.toml and Manifest.toml for the experiments section unchanged.